docs/platform/visual/index.md

```diff
@@ -1,133 +1,68 @@
+  and handle user interaction, 
+  dispatching [actions]({{site.refDocsUrlPattern | replace: '$', 'pentaho.visual.action'}}) and, 
+  for example, showing tooltips.
 
-# Bundling and provisioning
+The Visualization API is built on top of other Platform JavaScript APIs:
 
-The Pentaho platform is built on top of an OSGi container, so developers must provide their code as an OSGi bundle. 
-Additionally, the required client side dependencies must also be provided to the platform as bundles.
-
-The recommended way is to put the visualization bundle, its dependencies, 
-and corresponding feature definition together into a single KAR file.
-
-See [Bundling](bundling) for instructions.
-
-# Deploying the visualization
-
-The platform supports hot deployment: simply drop a bundle file in the deploy directory and 
-Apache Karaf will detect the file and try to deploy it.
+- The [Data API]({{site.refDocsUrlPattern | replace: '$', 'pentaho.data'}}) 
+  ensures seamless integration with data sources in the Pentaho platform, 
+  as well as with other client-side component frameworks.
 
-For PDI the Karaf folder is located in `system/karaf`. 
-On the Pentaho Server it's found within `pentaho-solutions/system/karaf`.
+- The [Type API]({{site.refDocsUrlPattern | replace: '$', 'pentaho.type'}}) 
+  provides to visualizations out-of-the-box features such as class inheritance, metadata support, configuration, 
+  validation and serialization.
 
-You can drop any KAR file, bundle or Feature file into the `karaf/deploy` folder. 
-It will be automatically installed and activated, even after restarts of the product. 
-Replacing a bundle or feature already in the deploy folder will reinstall it within OSGI. Deleting it will uninstall.
+- The [Core APIs](../../#core) provide to visualizations features such as localization, theming and 
+  services registration and consumption.
 
-If every thing went well you should now see your visualization in Analyzer and/or PDI.
+A set of stock visualizations is included, covering the most common chart types.
+Based on the [CCC](http://community.pentaho.com/ctools/ccc/) charting library, 
+they're customizable and extensible to fit your organization's desired look and feel.
 
-<img src="img/sample-bar-d3-analyzer.png" alt="Bar/D3 in Analyzer" style="width: 100%; max-width: 575px;"><br>
-*Your Bar/D3 walkthrough visualization in Analyzer*
+If you want to know more about the specifics of how Analyzer exposes the Visualization API, 
+read [Analyzer and the Visualization API](analyzer-viz-api).
 
-<img src="img/sample-bar-d3-pdi-menu.png" alt="Bar/D3 in PDI - menu" style="width: 40%; max-width: 274px;  min-width: 274px;">
-<img src="img/sample-bar-d3-pdi.png" alt="Bar/D3 in PDI" style="width: 60%;max-width: 583px;  min-width: 420px;"><br>
-*Your Bar/D3 walkthrough visualization in PDI*
+# Topics
+    
+1. [Creating a visualization](samples/bar-d3-sandbox)
+2. [Web-Packaging a visualization](web-packaging)
```

Comment:
I don't quite like the _sound_ of "Web-Packaging"... Can it just be "packaging"?